### ISSUE TITLE
Can't do Admin things when a player is offline

### ISSUE THREAD

**Issue by ZeekDaGeek** [LatvianModder/FTBUtilities#324]
Attempting to use /ftb chunks admin_unclaim_all <player> will fail when the user is offline allowing people to log in, claim chunks to grief and just not log in again in order to prevent you from ever fixing the issue.

```
java.lang.NullPointerException: EntityPlayer can't be null!
        at com.google.common.base.Preconditions.checkNotNull(Preconditions.java:229) ~[minecraft_server.1.10.2.jar:?]
        at com.feed_the_beast.ftbl.api_impl.ForgePlayer.getPlayer(ForgePlayer.java:187) ~[ForgePlayer.class:?]
        at com.feed_the_beast.ftbl.api_impl.ForgePlayer.isFake(ForgePlayer.java:194) ~[ForgePlayer.class:?]
        at com.feed_the_beast.ftbl.lib.cmd.CommandLM.getForgePlayer(CommandLM.java:80) ~[CommandLM.class:?]
        at com.feed_the_beast.ftbu.cmd.chunks.CmdAdminUnclaimAll.func_184881_a(CmdAdminUnclaimAll.java:38) ~[CmdAdminUnclaimAll.class:?]
        at net.minecraftforge.server.command.CommandTreeBase.func_184881_a(CommandTreeBase.java:147) ~[CommandTreeBase.class:?]
        at net.minecraftforge.server.command.CommandTreeBase.func_184881_a(CommandTreeBase.java:147) ~[CommandTreeBase.class:?]
        at com.feed_the_beast.ftbu.ranks.CmdTreeOverride.func_184881_a(CmdTreeOverride.java:50) ~[CmdTreeOverride.class:?]
        at net.minecraft.command.CommandHandler.func_175786_a(CommandHandler.java:104) [k.class:?]
        at net.minecraft.command.CommandHandler.func_71556_a(CommandHandler.java:83) [k.class:?]
        at net.minecraft.network.NetHandlerPlayServer.func_147361_d(NetHandlerPlayServer.java:913) [me.class:?]
        at net.minecraft.network.NetHandlerPlayServer.func_147354_a(NetHandlerPlayServer.java:892) [me.class:?]
        at net.minecraft.network.play.client.CPacketChatMessage.func_148833_a(SourceFile:37) [im.class:?]
        at net.minecraft.network.play.client.CPacketChatMessage.func_148833_a(SourceFile:9) [im.class:?]
        at net.minecraft.network.PacketThreadUtil$1.run(SourceFile:13) [fl$1.class:?]
        at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) [?:1.8.0_101]
        at java.util.concurrent.FutureTask.run(FutureTask.java:266) [?:1.8.0_101]
        at net.minecraft.util.Util.func_181617_a(SourceFile:45) [h.class:?]
        at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:668) [MinecraftServer.class:?]
        at net.minecraft.server.dedicated.DedicatedServer.func_71190_q(DedicatedServer.java:387) [ld.class:?]
        at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:613) [MinecraftServer.class:?]
        at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:471) [MinecraftServer.class:?]
        at java.lang.Thread.run(Thread.java:745) [?:1.8.0_101]
```

**Comment by linktlh** [comment]
first off hi Zeek, also we are having the same problem its a huge problem due to us needing to remove Claimed Chunks I'd love to see this fixed or another way for us to remove claimed chunks, (aka Bump)

**Comment by LatvianModder** [comment]
That's weird, it shouldn't crash. Try updating FTBLib&Utils to 3.2.9?

**Comment by linktlh** [comment]
It's not a crash it's what the console spews out.

**Comment by linktlh** [comment]
Bumpity bump bump. :3

**Comment by LatvianModder** [comment]
admin_unclaim_all will be replaced with simply unclaim_all

**Comment by LatvianModder** [comment]
Fixed in 3.4.0+
